lib/router/index.js

```diff
@@ -0,0 +1,51 @@
+const { Router } = require('express');
+const uuid = require('uuid/v4');
+
+const CaseRouter = require('./case');
+const Case = require('../db/case');
+
+module.exports = ({ hooks }) => {
+
+  const router = Router();
+
+  router.use((req, res, next) => {
+    res.respondWith = data => {
+      res.json({ data });
+    };
```

Comment:
I think this is how I should have done the "send-response" middleware in other services instead of avoiding early termination of the middleware stack. 

The problem is sending in a single location in code and not a common point in the stack.